src/sentry/runner/initializer.py

```diff
@@ -402,13 +402,19 @@ def validate_options(settings):
 def __model_unpickle_compat(model_id, attrs=None, factory=None):
     from django import VERSION
 
+    if attrs is not None or factory is not None:
+        metrics.incr("django.pickle.loaded_19_pickle.__model_unpickle_compat", skip_internal=False)
```

Comment:
Remove the skip_internal flag plz. We don’t need it.

Reply:
Should this metric be sampled?

Reply:
Don't want to sample for the moment since I don't know how often this is even getting hit. If it turns out to get hit a lot I'll enable sampling. I assume the main factor here is cost?

Reply:
It samples by default at our sample rate, so you'd have to explicitly set the sample rate to 1 to opt out of it.